backend/canisters/group/impl/src/lib.rs

```diff
@@ -100,6 +104,45 @@ impl RuntimeState {
         }
     }
 
+    pub fn queue_access_gate_payments(&mut self, gate: PaymentGate) {
+        // The amount available is the gate amount less the approval fee and the transfer_from fee
+        let amount_available = gate.amount - 2 * gate.fee;
+        // Queue a payment to each owner less the fee
+        let owners: Vec<UserId> = self
+            .data
+            .chat
+            .members
+            .iter()
+            .filter(|m| matches!(m.role.value, GroupRoleInternal::Owner))
+            .map(|m| m.user_id)
+            .collect();
+
+        let owner_count = owners.len() as u128;
+        let owner_share = (amount_available * 4 / 5) / owner_count;
```

Comment:
I thought they would take 80% of the amount before any fees are taken, then the OC treasury would pay all the fees. This makes the users receive nice numbers

Reply:
There could theoretically be 100 owners and we agreed it made sense for each owner to pay their own fee